packages/core/test/invoke.test.ts

```diff
@@ -2050,6 +2053,116 @@ describe('invoke', () => {
     });
   });
 
+  describe('with behaviors', () => {
```

Comment:
we could use at last some basic test for spawning behaviors - unless I've missed it only tests using `invoke` have been introduced.

Reply:
Done here: c30c0386cd87aa9ff7d73cbe2da703a9b3d5b5cd